angelsindustries/prototypes/overrides/global-tech-base-packs.lua

```diff
@@ -60,6 +60,8 @@ if angelsmods.industries.tech then
   pack_replace("angels-components-construction-2", "green", "red")
   pack_replace("angels-components-cabling-2", "green", "red")
   pack_replace("angels-components-mechanical-2", "green", "red")
+  OV.remove_prereq("circuit-network", "tech-green-packs")
```

Comment:
For some reason, replacing green with red packs wasn't enough; the circuit network tech still depended on green pack tech.

Reply:
Because that's how it is in the base game as well...
https://github.com/wube/factorio-data/blob/29efc79aa0505e08c461c19b86e1085f3cf5f101/base/prototypes/technology.lua#L5541-L5590

I don't think it's a bad idea to move it to red science, however we do have to double check that it has the correct prerequisites according to the ingredients